Plugins/DeleteUnusedBranches/GitBranchOutputCommandParser.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+namespace DeleteUnusedBranches
+{
+    public class GitBranchOutputCommandParser
+    {
+        public IEnumerable<string> GetBranchNames(string commandOutput)
+        {
+            if (commandOutput != null)
+            {
+                foreach (var line in commandOutput.Split('\n'))
+                {
+                    if (!string.IsNullOrWhiteSpace(line))
```

Comment:
Please amalgamate into the `foreach`.

Reply:
Can you please elaborate on this?

Do you mean something like:
```
foreach (var line in commandOutput.Split('\n').Where(x => !string.IsNullOrWhiteSpace(x)))
{
 ...
}
```

I don't think it's actually better than the original in terms of readability...